README.rst

```diff
@@ -1121,6 +1121,10 @@ Choose from the list of available rules:
 
   PHPUnit annotations should be a FQCNs including a root namespace.
 
+* **php_unit_internal_class**
+
+  All PHPUnit test classes should be marked as internal.
```

Comment:
I would either not mark with `@internal` phpunit classes that are abstract, or add config option to control it.
Abstract `*TestCase` are often use as extension point for external ppl, same like in our repo.

Reply:
I've added the options
- `normal`
- `final`
- `abstract`

Normal meaning non final, non abstract.